src/aura/HH_Container/HH_Container.cmp

```diff
@@ -56,14 +56,13 @@
                         <nav role="navigation" class="slds-p-bottom--x-small">
                             <p id="bread-crumb-label" class="slds-assistive-text">You are here:</p>
                             <ol class="slds-breadcrumb slds-list--horizontal" aria-labelledby="bread-crumb-label">
-                                <li class="slds-breadcrumb__item slds-text-heading--label"><a class="slds-type-focus" href="{!'/' + v.hhTypePrefix}">{!$Label.c.lblHousehold}</a></li>
```

Comment:
why did you remove slds-type-focus?  I believe that was giving me the hover hyperlink underline that no longer works.